src/index.ts

```diff
@@ -29,6 +29,8 @@ export * from "./constants"
 
 export * from "./url_utils"
 
+export * from "./search"
```

Comment:
Could we export just what we use, rather than everything?

For example, it looks like `buildSuggestQuery` and `buildDefaultSort` are exported from `search.ts` for testing purposes. But if we're not actually consuming those other packages, maybe we could leave them as implementation details for now and here just export what we need

```suggestion
export { buildLearnQuery, buildSearchQuery, buildChannelQuery } from "./search"
export type { SearchQueryParams } from "./search"
```
(Maybe there are more?)